anyway/infographics_utils.py

```diff
@@ -55,6 +97,54 @@ def serialize(self):
         return output
 
 
+class RequestParams:
+    """
+    Input for infographics data generation, per api call
+    """
+    def __init__(self,
+                 news_flash_id: int,
+                 location_text: str,
+                 location_info: Optional[dict],
+                 resolution: Dict,
+                 gps: Dict,
+                 start_time: datetime.date,
+                 end_time: datetime.date):
+        self.news_flash_is = news_flash_id
+        self.location_text = location_text
+        self.location_info = location_info
+        self.resolution = resolution,
+        self.gps = gps
+        self.start_time = start_time
+        self.end_time = end_time
```

Comment:
Again, consider using data class to avoid this boilerplate, this init function is redundant

Reply:
Correct. I wasn't aware of dataclasses.